exercism/python/pangram/pangram.py

```diff
@@ -6,3 +114,26 @@ def is_pangram(sentence):
     return (
         len(set(letter.lower() for letter in re.findall(r"[A-Za-z]", sentence))) == 26
     )
+
+"""
+Pros: Efficient: good use of time/space complecity
```

Comment:
For the pro, I would say concise and taking advantage of built-in libraries

Reply:
not sure this is really good complexity wise because regexes are awful for performance